example/spec/parallel/ingest/ingestFromPdrSpec.js

```diff
@@ -147,40 +155,51 @@ describe('Ingesting from PDR', () => {
     let queuePdrsOutput;
 
     beforeAll(async () => {
-      workflowExecution = await buildAndExecuteWorkflow(
-        config.stackName,
-        config.bucket,
-        workflowName,
-        undefined,
-        provider,
-        undefined,
-        { provider_path: testDataFolder }
-      );
-
-      queuePdrsOutput = await lambdaStep.getStepOutput(
-        workflowExecution.executionArn,
-        'QueuePdrs'
-      );
+      try {
+        workflowExecution = await buildAndExecuteWorkflow(
+          config.stackName,
+          config.bucket,
+          workflowName,
+          { name: addedCollection.name, version: addedCollection.version },
```

Comment:
Adding this collection was required as other changes in the PR removed the "default" behavior that resulted in a collection of "default____default" being applied when it was missing from the message.